components/ui-shell/index.ts

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) IBM, Corp. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import 'carbon-web-components/es/components/ui-shell/index.js';
+import './header/index.js';
+import './side-nav/index.js';
+
+import { qiskitLogoIcon } from '../icons/qiskit-logo.js';
+import { userIcon } from '../icons/user.js';
+import styles from './index.scss';
+
+interface NavItem {
+  label: string;
+  url?: string;
+  children?: NavItem[];
+  segment?: SegmentData;
+}
+
+interface SegmentData {
+  cta: string;
+  location: string;
+}
+
+enum Variant {
+  DEFAULT = '',
+  WITH_ACCOUNT = 'with-account',
+}
+
+@customElement('qiskit-ui-shell')
+export class QiskitUIShell extends LitElement {
+  static styles = [styles];
+
+  @property({ type: String })
+  variant: Variant = Variant.DEFAULT;
+
+  private _NAV_ITEMS: NavItem[] = [
+    {
+      label: 'Overview',
+      url: 'https://qiskit.org/overview/',
+    },
+    {
+      label: 'Learn',
+      url: 'https://qiskit.org/learn/',
+    },
+    {
+      label: 'Community',
+      children: [
+        {
+          label: 'Events',
+          url: 'https://qiskit.org/events/',
+        },
+        {
+          label: 'Advocates',
+          url: 'https://qiskit.org/advocates/',
+        },
+      ],
+    },
+    {
+      label: 'Documentation',
+      url: 'https://qiskit.org/documentation/',
+    },
+  ];
+
+  render() {
+    return html`
+      <bx-header aria-label="Qiskit" class="qiskit-header">
+        <qiskit-header-name href="https://qiskit.org/">
+          ${qiskitLogoIcon}
+        </qiskit-header-name>
+        <qiskit-header-nav menu-bar-label="Qikskit">
+          ${this._NAV_ITEMS.map((item) => {
```

Comment:
Non blocker, just a preference, so if you prefer your way, keep it :) : I prefer to create functions for logic so the template gets shorter and, for me, it's easy to read

Reply:
done with https://github.com/Qiskit/web-components/pull/139/commits/05577edf60029402d531543b1621f93e9aa1f0d9
nice and clean 😌 thank you!